src/huggingface_hub/keras_mixin.py

```diff
@@ -39,10 +39,13 @@ def _extract_hyperparameters_from_keras(model):
     return hyperparameters
 
 
-def _parse_model_history(model):
+def _parse_model_history(model, save_directory):
```

Comment:
I'm not in love with the fact this history writing is happening so deep in the logic here. `save_pretrained_keras` -> `_create_model_card` -> `_write_metrics` -> `_parse_model_history` is so deep in the code to just save `model.history.history` as a json file. 

I don't want to block on this, but just wanted to point that out so we remember to put this in a nicer spot in future refactors.